src/components/PageBanner/PageBanner.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "~styles/variables.scss";
+
+.pageBanner {
+  background-color: $brand-lightGrey;
```

Comment:
Definitely agree here - will have @corwinharrell take a look and see the deployed version (will put this up on staging soon)

Reply:
@Calpoog I have to admit that I think the color palette that has been used in our design mocks and the one that we have defined in prod have diverged a bit since our original design exploration (As things tend to do). Feel free to introduce a new grey value for now if there isn't one that exists that's close enough, and I'll try and address the inconsistencies at some point.